search/search-domain/src/main/java/io/camunda/util/ExceptionUtil.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.util;
+
+public class ExceptionUtil {
```

Comment:
🔧 We only collect error messages here, so let's maybe name this accordingly, like `ErrorMessages`. We have a similar thing in the REST gateway.